updateutils/update.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) 2021 - Jojo#7791
+# Licensed under MIT
+
+import asyncio
+import asyncio.subprocess as asp
+import os
+import sys
+from datetime import datetime
+
+import aiohttp
+import discord
+from jojo_utils import __version__ as jojoutils_version
+from redbot.core import commands
+from redbot.core.bot import Red
+from redbot.core.utils.predicates import MessagePredicate
+import logging
+
+
+log = logging.getLogger("red.JojoCogs.updateutils")
+VERSION_URL = r"https://raw.githubusercontent.com/Just-Jojo/jojoutils/master/version"
+
+
+class UpdateUtils(commands.Cog):
+    """A developer tool to update Jojo's utils"""
+
+    __authors__ = [
+        "Jojo#7791",
+    ]
+    __version__ = "1.0.0"
+
+    def __init__(self, bot: Red):
+        self.bot = bot
+        self.session = aiohttp.ClientSession()
+        sep = os.path.sep
+        # This section is a modified version of Jack's shell's `get_env` function
+        # Which is copyrighted under the Apache-2.0 License.
+        # https://github.com/jack1142/JackCogs/blob/v3/shell/utils.py#L30
+        if sys.platform == "win32":
+            self.path = f"{sys.prefix}{sep}Scripts{sep}python.exe"
+        else:
+            self.path = f"{sys.prefix}{sep}bin{sep}python3.8"
```

Comment:
```suggestion
        self.path = Path(sys.executable).resolve()
```

Reply:
``sys.executable`` is essentially exactly the same as what you're doing here, fyi